src/k8s/pkg/k8sd/features/cilium/cleanup.go

```diff
@@ -0,0 +1,22 @@
+package cilium
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/canonical/k8s/pkg/snap"
+)
+
+func CleanupNetwork(ctx context.Context, snap snap.Snap) error {
+	os.Remove("/var/run/cilium/cilium.pid")
```

Comment:
it is preferred to either log the error, or be explicit that you ignore it, by means of `_ = os.Remove("/var/run/cilium/cilium.pid")`. but we don't have good logging practices in place, so that's acceptable as is

Reply:
Agreed, I wasn't sure how we do logging across the code so open to suggestions there if not I can make it explicit the way you've described.